backend/typescript/services/interfaces/schedulingService.ts

```diff
@@ -34,20 +34,6 @@ interface ISchedulingService {
    */
   getSchedulings(): Promise<Array<SchedulingDTO>>;
 
```

Comment:
Can we add in the correct `sendScheduledDonationEmail` into the interface here

Reply:
Done